### PR TITLE
feat(mixin): Add grafana mixins support for postgres exporter

### PR DESCRIPTION
**Pull request by kavirajk**
- Add grafana mixin support for postgres_exporter
- More about mixins here  https://github.com/monitoring-mixins/docs.

### CODE REVIEW

File: postgres_mixin/dashboards/postgres-overview.json [hunk 1]
```diff
@@ -0,0 +1,1321 @@
...
+      "renderer": "flot",
+      "seriesOverrides": [],
+      "spaceLength": 10,
+      "stack": false,
+      "steppedLine": false,
+      "targets": [
+        {
+          "alias": "fetched",
+          "dsType": "prometheus",
+          "expr": "sum(irate(pg_stat_database_tup_fetched{datname=~\"$db\",instance=~\"$host\"}[5m]))",
```
Comment (tomwilkie) [inline postgres_mixin/dashboards/postgres-overview.json:104]: Can you setup template variables for datasource, job and instance?  That should make the dashboard more generally applicable.

**Comment by jeschkies** [comment]
@wrouesnel, what do you think? Could this be merged :slightly_smiling_face:

**Comment by SuperQ** [comment]
Would you please rebase this against the latest revision? It should kick off updated tests.

**Comment by kavirajk** [comment]
@SuperQ rebased!

**Comment by matthiasantierens** [comment]
Did you already look at this dashboard? (https://github.com/lstn/misc-grafana-dashboards) It looks more complete :smiley:, Could we combine them?

**Review by SuperQ — APPROVED** [review]
LGTM, Thanks! More mixins!

**Pull request merged**